pkg/src/src/main/scala/edu/berkeley/cs/amplab/sparkr/SparkRSQL.scala

```diff
@@ -0,0 +1,12 @@
+package edu.berkeley.cs.amplab.sparkr
+
+import org.apache.spark.SparkContext
+import org.apache.spark.sql.{SQLContext}
+import org.apache.spark.sql.types.{StructType}
+
+object sparkRSQL {
```

Comment:
Fixed.  Also tweaked my settings in Sublime so that 2-space indents should all start happening by default :)